docs/cli/csv-parser-state.md

```diff
@@ -0,0 +1,163 @@
+# CSV Parser State
+
+Convert [commercetools states](https://docs.commercetools.com/http-api-projects-states.html#state) CSV data to JSON. See example below for CSV format, sample response and usage.
+
+## Usage
+`npm install @commercetools/csv-parser-state --global`
+
+### CLI
+```
+Usage: csv-parser-state.js [options]
+Package to parse states from CSV to JSON
+
+Options:
+  --help, -h                 Show help                                 [boolean]
+  --version, -v              Show version number                       [boolean]
+  --projectKey, -p           API project key                            [string]
+  --apiUrl                   The host URL of the HTTP API service
+                             [string] [default: "https://api.commercetools.com"]
+  --authUrl                  The host URL of the OAuth API service
+                            [string] [default: "https://auth.commercetools.com"]
+  --accessToken              CTP client access token
+                             Required scopes: ['view_orders']         [string]
+  --input, -i                Path to CSV file.                [default: "stdin"]
+  --output, -o               Path to output JSON file
+                                                    [string] [default: "stdout"]
+  --delimiter, -d            Used CSV delimiter.                  [default: ","]
+  --multiValueDelimiter, -m  Used CSV delimiter in multiValue fields.
+                                                                  [default: ";"]
+  --continueOnProblems, -c   Flag if parsing should continue if module
+                             encounters an error.     [boolean] [default: false]
+  --logLevel                 Logging level: error, warn, info or debug
+                                                      [string] [default: "info"]
+  --prettyLogs               Pretty print logs to the terminal         [boolean]
+  --logFile                  Path to file where logs should be saved
+                                   [string] [default: "csv-parser-state.log"]
+```
+
+#### Info on flags
+- The `--projectKey` is only required if states have a `transitions` field containing one or more states that need to be resolved, otherwise it can be omitted. More info in the **Examples** below
```

Comment:
is there a default if `projectKey` isn't provided?

Reply:
No there isn't. It defaults to undefined